CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 
-## 1.0
+## 1.1.0 – [diff](https://github.com/openfisca/openfisca-france/compare/1.0...1.1.0)
```

Comment:
Don't get lost in qualifying “new functionality” or not. The algorithm is downward, not upward:

- Is there a single possible existing call to the public API that could behave differently than was expected before the change? If yes, it's a major bump. (here, based on @fpagnoux remark, it did)
- Is there a new possible call to the public API that was not expected before? If yes, it's a minor bump.
- Is there a call to the public API that was expected to work but in actuality did not, and now does when applying the changeset? Philosophically, it's a bug fix, but in terms of SemVer it's a patch bump.
- Is there some internal refactor that doesn't change anything? You may release a patch bump. But I would recommend you simply release a _pre-release patch bump_, and delay release until you actually have something to your users.

Remember: the only use of SemVer is presenting metadata on the public API and easing dependency management. Here is a [cheat sheet](https://speakerdeck.com/mattisg/git-session-2-strategies).